ads/_ping_.js

```diff
@@ -21,7 +21,21 @@ import {dev} from '../src/log';
  * @param {!Object} data
  */
 export function _ping_(global, data) {
+  //validateData(data, [], ['adWidth', 'adHeight', 'valid']);
   global.document.getElementById('c').textContent = data.ping;
+  global.ping = Object.create(null);
```

Comment:
I don't know much about the purpose of the `_ping_` file, but if the `.ping` object is purely for testing purposes, we need to do it in a way that gets DC'd in prod builds.

One way might be moving the creation and the logic related to global.ping to a function like `setupPingForTesting` and call that function as part of the integration test setup.